### DISCUSSION TITLE
Install older version of MPV using Homebrew

### DISCUSSION THREAD

**Discussion by BlairSibley** [Homebrew/discussions#49] [Everyday usage] [answered]
This may be a really obvious question, but I am trying to install mpv in Homebrew. Because of an incompatibility issue with another formula, I need to install mpv-0.31.0 instead of the current version (0.32.0). So when I run the following code:

`brew cask install mpv`

It only gives me the latest version. I see it getting the tar.gz from this link: https://laboratory.stolendata.net/~djinn/mpv_osx/.

So there has to be a way for me to point it towards version 0.31.0, which is already there. I'm a total noob at coding, so apologies if this is something very obvious. I searched the Internet several times and tried using the following:

`brew cask install mpv@0.31.0`

Which didn't work unfortunately. So I'm not sure what to try next. Thanks in advance!

**Comment by gromgit** [comment, marked as answer]
I don't think versioning is supported for casks, and "incompatibility issue with another formula" sounds like you're focusing on the command-line `mpv` binary anyway, so installing the corresponding _formula_ may work better for you, and `brew` does support extracting and installing older formulae versions. You'd just have to create your own tap to host them:
```
brew tap-new gromgit/versions
brew extract --version 0.31.0 mpv gromgit/versions
brew install gromgit/versions/mpv@0.31.0
```

**Reply by BlairSibley** [reply]
Thank you, I will try this
